app/controllers/api/posts_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  class Api::PostsController < ApiController
+    # load_and_authorize_resource
```

Comment:
- [ ] Please remove commented code if not necessary!

Reply:
thanks! It's done